e2e/auth.cleanup.ts

```diff
@@ -1,58 +1,41 @@
-import { Page, test as cleanup, expect } from '@playwright/test'
-import { ADMIN_STORAGE_STATE, USER_STORAGE_STATE } from '../playwright.config'
-import { getEndpoint } from './helpers/http'
+import { Page, test as cleanup } from '@playwright/test'
 import {
-  deleteTestUser,
-  getAccessTokenFromCookie,
+  cleanupTestUser,
   getUserIdFromLocalStorage,
   goToDashboard,
   logoutTestUser,
 } from './helpers/testUser'
+import { userConfigs } from './helpers/userConfig'
 
-let adminPage: Page
-let userPage: Page
+for (const { testName, storageStatePath, user } of userConfigs) {
```

Comment:
Loop through a config file to create and authenticate test users, which will hopefully make it easier to create new users in the future